src/components/logs/history/glucose.tsx

```
@@ -0,0 +1,103 @@
+            </div>
+        </div>
+    )
+}
+
+const GlucoseHistory = ({ timePeriod }: { timePeriod: string }) => {
+    const { data, isLoading: isGlucoseLogsLoading } = api.glucose.getLogs.useQuery()
+
+    const [glucoseLogs, setGlucoseLogs] = useState(data)
+    const pathname = usePathname()
+
+    useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: data

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)